### PR TITLE
Topic ui

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
screen shot 2016-07-18 at 5 13 31 pm

**Comment by robbiecarlton** [comment]
depends on https://github.com/Hylozoic/hylo-node/pull/149

**Comment by robbiecarlton** [comment]
@levity - good to merge?

File: src/containers/tag/TagPosts.js [hunk 1]
```diff
@@ -57,12 +60,20 @@ class TagPosts extends React.Component {
       return <div className='loading'>Please wait...</div>
     }
 
+    const { owner, followers } = tag
+
```
Comment (levity) [inline src/containers/tag/TagPosts.js:64]: no need for these extra newlines

File: src/containers/tag/TagPosts.js [hunk 2]
```diff
@@ -92,3 +103,15 @@ export default compose(
...
   })
 )(TagPosts)
+
+const Followers = ({followers}) => {
+  return <div className='followers'>
+    <span>
+      Followed by
+    </span>
+    {followers.slice(0, 3).map(follower => <Avatar key={follower.id} person={follower} />)}
+    {followers.length > 3 && <Dropdown toggleChildren={<span className='plus-button'><span className='content'>+{followers.length - 3}</span></span>}>
```
Comment (levity) [inline src/containers/tag/TagPosts.js]: break up this long line pls

**Comment by levity** [comment]
i think some of that needs to get hidden on mobile? want to add that and attach a screenshot of mobile?

**Comment by robbiecarlton** [comment]
screen shot 2016-07-19 at 11 44 27 am

**Comment by levity** [comment]
:+1:

**Pull request merged**